src/pages/settings/tabs/UserManagementSettings.tsx

```diff
@@ -133,15 +135,15 @@ function UserManagementSettings() {
 
   const changeAvatar = (avatar: File | undefined) => {
     if (!avatar) return;
-    editAvatar({ avatar, userId: selectedUser.ID })
-      .then(() => toast.success('', 'Avatar updated!'))
+    fileToDataUrl(avatar)
```

Comment:
If we don't have a changeAvatar endpoimt, we don't need these 2 functions right?
When we save the whole page it should be changed along with other user details

Reply:
As long as we can somehow convert the file to a data url and somehow reset it to an empty string, then no, we don't need these functions.

Reply:
Also, if we plan to use a client-side cropping util of some sorts, then can this function be used to take the file and give it to the util (that hopefully opens up as a modal of some sorts).

Reply:
nvm, I read the code wrong